src/FastState.h

```diff
@@ -56,12 +61,18 @@ class FastState {
     FullBoard board;
 
     float m_komi;
+    float m_stm_komi;
+    float m_opp_komi;
     int m_handicap;
     int m_passes;
     int m_komove;
     size_t m_movenum;
     int m_lastmove;
 
+    bool pos_komi() { return (get_to_move() == FastBoard::BLACK && (m_opp_komi > 7.5 || m_stm_komi > 7.5)) || (get_to_move() == FastBoard::WHITE && (m_opp_komi < -7.5 || m_stm_komi < -7.5)); }
```

Comment:
Line length. Also in general put as much functions in the .cpp as possible, and let the compiler figure out the inlining.